CHANGELOG.asciidoc

```diff
@@ -4,6 +4,21 @@
 :pull: https://github.com/elastic/apm-server/pull/
 
 
+=== APM Server version HEAD
+https://github.com/elastic/apm-server/compare/9e0a1e281e56044745f1a4f18dcbf00a7fa03683\...master[View commits]
+
+==== Breaking changes
+
+==== Bug fixes
+
+- Don't augment payloads with invalid IPs {pull}923[923], {pull}927[927].
```

Comment:
this bug never made it to a production system, right? - it has been created and fixed in 6.3
do we need a changelog?